.github/workflows/clear-cache.yml

```
@@ -0,0 +1,31 @@
+name: Clear branch cache
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch_number:
+        description: "Branch number"
```

```suggestion
        description: "PR number"
```

The cache is associated with the PR number. So calling it PR number makes it clearer, imo

---

agree, although it's kinda WIP :)

---

Yes, yes, it was not a formal review, just a suggestion